src/modules/notifications/components/NotificationItem.js

```diff
@@ -0,0 +1,88 @@
+import { BaElement } from '../../BaElement';
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { $injector } from '../../../injection';
+import { LevelTypes } from '../../../store/notifications/notifications.reducer';
+import css from './notificationItem.css';
+
+export const NOTIFICATION_AUTOCLOSE_TIME_NEVER = 0;
```

Comment:
Didn't we prefer  the `First_Letter_Capitalized_Snake_Case_Style` for private constants 🤔

(see also NotificationPanel.js line 9)

Reply:
..but this constant have a public meaning

Reply:
so it must be: 
static get NOTIFICATION_AUTOCLOSE_TIME_NEVER(){
    return 0;
}

right?

Reply:
> ..but this constant have a public meaning

Ah, you're right. We can leave this here, sorry